wuvt/templates/index.html

```diff
@@ -6,7 +6,8 @@
 <article>
     <header>
         <h2><a href="{{ url_for('.article', slug=article.slug) }}">{{ article.title }}</a></h2>
-        <h3>written by {{ article.author.name }} on <time datetime="{{ article.datetime|isodatetime }}" data-format="MMMM D, YYYY, HH:mm">{{ article.datetime|datetime("%B %d, %Y, %H:%M") }}</time></h3>
+        <h3>written by {{ article.author.name }} on <time datetime="{{ article.datetime|isodatetime }}" data-format="MMMM D, YYYY, HH:mm">{{ article.datetime|datetime("%B %d, %Y, %H:%M") }}</time>
+		{% if article.pinned_article -%}article pinned{% endif -%}</h3>
```

Comment:
Personally I don't think there's a good reason to expose this implementation detail to the user.

Reply:
I see it less as exposing implementation details and more of the little tack icon next to a pinned tweet that lets you know this post is special that's why it's here. But "article pinned" is a pretty technical way of describing that I agree. Maybe it could say "notice"? Or just not there at all is fine too that is a good point.

Reply:
Maybe just an icon, "pinned", or "notice" would be good

Reply:
yeah. that would be next step. I agree. I just wanted to bash this out first.